changelog/@unreleased/pr-479.v2.yml

```
@@ -0,0 +1,5 @@
+type: fix
+fix:
+  description: [Gradle JDK Automanagement] Install only used JDKs
```

```suggestion
  description: '[Gradle JDK Automanagement] Install only used JDKs'
```

Changelog is failing otherwise